code/game/machinery/doors/airlock.dm

```diff
@@ -1198,6 +1200,28 @@
 					to_chat(user, "<span class='warning'>Despite your attempts, [src] refuses to open.</span>")
 			prying_so_hard = FALSE
 
+/obj/machinery/door/airlock/proc/open_with_fake_card(obj/card, mob/user)
+	add_fingerprint(user)
+	if(operating || welded || locked)
+		return FALSE
+	var/obj/item/card/id/fake_card/fake = card
+	if(fake.uses)
+		if(check_access_list(fake.access))
+			visible_message("<span class='warning'>[user] starts fumbling at [src] with a piece of paper!</span>")
```

Comment:
Should have a message sent to the user as well, stating that "you start swiping the card at the door" or somethijng along those lines.
I believe self_message is an argument for visible_message